source/main.cpp

```diff
@@ -294,6 +324,11 @@ int main(int argc, char *argv[])
         LOG_WARN(TAG, "Unable to append current working directory to PATH environment variable.");
     }
 
+    if (!init(argc, argv))
```

Comment:
Add a comment or update the function name so that it is easy to understand the meaning of the following code statements.